.github/ISSUE_TEMPLATE/bug_report.md

```diff
@@ -1,3 +1,12 @@
+---
+name: Bug report
+about: Create a report to help us improve.
+title: ''
+labels: ''
+assignees: ''
+
```

Comment:
but i do'nt know what this part is about though

Reply:
That is Githubs new stuff, to auto assign or lable or title... Can be left blank

Reply:
ok cool